tests/conftest.py

```diff
@@ -34,11 +34,11 @@ def _get_client(cls=None, **kwargs):
 
     params = {'startup_nodes': [{'host': '127.0.0.1', 'port': 7000}], 'socket_timeout': 10, 'decode_responses': False}
     params.update(kwargs)
-    return StrictRedisCluster(**params)
+    return cls(**params)
```

Comment:
@Grokzen Thanks for the fix. The UT fails because of this line and it was a bug before that providing a `cls` argument doesn't have any effect.